docs/guide/snaps.md

```diff
@@ -30,55 +31,38 @@ Among other things, SES allows us to restrict access to global JavaScript APIs a
 
 ## Getting Started
 
-There are many simple snaps available for you to learn from and begin to understand what your own snap needs. The following tutorial will walk you through a "Hello, World!" snap.
-
 ### Prerequisites
+- [Node.js 16.0](https://nodejs.org/) or later.
 
-- Node.js
-  - We recommend using [nvm](https://github.com/nvm-sh/nvm). If you don't want to use nvm, your local Node version needs to be greater than or equal to the one listed [here](https://github.com/MetaMask/template-snap/blob/main/.nvmrc).
-- git
-- [yarn v1](https://classic.yarnpkg.com)
-- Up-to-date Chromium or Firefox browser
-
-### Installing MetaMask Flask
-
-[MetaMask Flask](https://metamask.io/flask) is a separate extension for developers that provides access to additional unstable APIs. The goal of Flask is to maximize developer control, so that we can learn the full extent of what developers want to do with MetaMask and incorporate those lessons into the stable MetaMask distribution. Snaps is the first feature to be released on Flask.
-
-- First, in a Firefox or Chromium browser, create a new browser profile or disable any existing installed versions of MetaMask.
-  - Running multiple instances of MetaMask in the same browser profile will break dapp interactions.
-- Next, download [Flask](https://metamask.io/flask) and add it your chosen browser.
+### Install MetaMask Flask
 
-### Serving a Snap to Your Local Environment
+To interact with (your) Snaps, you will need to install [MetaMask Flask](https://metamask.io/flask/), a canary distribution for developers that provides access to upcoming features. 
 
-With Flask in your browser, it's time to plug something into it. If you don't want to download any code for the moment, check out the example snap [deployed here](https://metamask.github.io/template-snap/) and follow the instructions below under 'Interacting with the Snap'.
+Make sure to install Flask in a new browser profile or disable any existing installed versions of MetaMask. Running multiple instances of MetaMask in the same browser profile will break dapp interactions.
 
-If you're ready to jump in and see how a Snap looks under the hood, follow the steps below!
+### Quick start using our template
+Get started using our template built with TypeScript and React. Clone the repository via the command line or [via GitHub](https://github.com/MetaMask/template-snap-monorepo/generate).
+```shell
+git clone git@github.com:MetaMask/template-snap-monorepo.git    
+```
 
-Open up a new terminal window, and do the following:
 
-- Clone the [template snap](https://github.com/MetaMask/template-snap) repo.
-- `cd` into the repo, and run `nvm use`.
```

Comment:
we should keep the nvm use step

Reply:
We don't require users to use node through nvm though? But we could advice them to if we feel it's essential

Reply:
We designed the template to work against a certain Node version, and `nvm` makes it easy to install that version and get going, so in the spirit of streamlinedness I think it's good to have a recommendation for `nvm`.

Reply:
especially as a noob, nvm errors can be bewildering, I agree with including some guard rails here